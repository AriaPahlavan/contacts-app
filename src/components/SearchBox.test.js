import {shallow} from 'enzyme';
import React from 'react';
import SearchBox from './SreachBox';

it('ensures SearchBox rendered like snapshot', () => {
  expect(shallow(<SearchBox />)).toMatchSnapshot();
});
