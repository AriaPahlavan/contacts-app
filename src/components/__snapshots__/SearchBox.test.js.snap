// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ensures SearchBox rendered like snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SreachBox />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <input
        aria-label="Search Kitties"
        className="pa3 ma2 ba b--green bg-lightest-blue"
        id="sb"
        onChange={undefined}
        placeholder="search kitties"
        type="search"
/>,
      "className": "pa2",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "aria-label": "Search Kitties",
        "className": "pa3 ma2 ba b--green bg-lightest-blue",
        "id": "sb",
        "onChange": undefined,
        "placeholder": "search kitties",
        "type": "search",
      },
      "ref": null,
      "rendered": null,
      "type": "input",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <input
          aria-label="Search Kitties"
          className="pa3 ma2 ba b--green bg-lightest-blue"
          id="sb"
          onChange={undefined}
          placeholder="search kitties"
          type="search"
/>,
        "className": "pa2",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "aria-label": "Search Kitties",
          "className": "pa3 ma2 ba b--green bg-lightest-blue",
          "id": "sb",
          "onChange": undefined,
          "placeholder": "search kitties",
          "type": "search",
        },
        "ref": null,
        "rendered": null,
        "type": "input",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getSnapshotBeforeUpdate": true,
        },
      },
    },
    "attachTo": undefined,
    "hydrateIn": undefined,
  },
}
`;
